Pose3_TEST.py fails in arm64 6.9.0~pre1

## Environment
* OS Version: Ubuntu Focal, arm64
* Source or binary build? during package creation

The [debbuilder job](https://build.osrfoundation.org/job/ign-math6-debbuilder/1084/console) from nightly today reports:
```
110/138 Test #110: Pose3_TEST.py ..........................***Failed    1.29 sec
......E
======================================================================
ERROR: test_stream_out (__main__.TestPose3)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/var/lib/jenkins/workspace/ign-math6-debbuilder/build/ign-math/src/python/Pose3_TEST.py", line 145, in test_stream_out
    self.assertAlmostEqual(str(p), "0.1 1.2 2.3 0 0.1 1")
  File "/usr/lib/python3.8/unittest/case.py", line 943, in assertAlmostEqual
    diff = abs(first - second)
TypeError: unsupported operand type(s) for -: 'str' and 'str'

----------------------------------------------------------------------
Ran 7 tests in 0.028s

FAILED (errors=1)
```

Seems to happen when "subtracting a string from another that contains numbers in both strings" which seems to me the thing what we are doing here. Why is not failing in other arches?

## Description
* Expected behavior: test to pass
* Actual behavior: fails making the packaging job to fail

the test failure is more clear now:

~~~
110/138 Test #110: Pose3_TEST.py ..........................***Failed    0.10 sec
......F
======================================================================
FAIL: test_stream_out (__main__.TestPose3)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/jenkins/workspace/ign-math6-debbuilder/build/ignition-math-6.9.1~pre1/src/python/Pose3_TEST.py", line 145, in test_stream_out
    self.assertEqual(str(p), "0.1 1.2 2.3 0 0.1 1")
AssertionError: '0.1 1.2 2.3 -0 0.1 1' != '0.1 1.2 2.3 0 0.1 1'
- 0.1 1.2 2.3 -0 0.1 1
?             -
+ 0.1 1.2 2.3 0 0.1 1


----------------------------------------------------------------------
Ran 7 tests in 0.001s

FAILED (failures=1)
~~~

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -15,7 +15,7 @@ find_package(ignition-cmake2 2.8.0 REQUIRED)
 # Configure the project
 #============================================================================
 set (c++standard 17)
-ign_configure_project(VERSION_SUFFIX pre1)
+ign_configure_project(VERSION_SUFFIX pre2)
 
 #============================================================================
 # Set project-specific options

--- a/Changelog.md
+++ b/Changelog.md
@@ -7,6 +7,9 @@
 1. Avoid assertAlmostEqual for python strings
     * [Pull request #255](https://github.com/ignitionrobotics/ign-math/pull/255)
 
+1. Pose3_TEST.py: use 0.01 (not 0) in string test
+    * [Pull request #257](https://github.com/ignitionrobotics/ign-math/pull/257)
+
 ## Ignition Math 6.9.0 (2021-09-28)
 
 1. Volume below a plane for spheres and boxes

--- a/src/python/Pose3_TEST.py
+++ b/src/python/Pose3_TEST.py
@@ -141,8 +141,8 @@ class TestPose3(unittest.TestCase):
         self.assertTrue(pose.rot() == Quaterniond(1, 0, 0))
 
     def test_stream_out(self):
-        p = Pose3d(0.1, 1.2, 2.3, 0.0, 0.1, 1.0)
-        self.assertEqual(str(p), "0.1 1.2 2.3 0 0.1 1")
+        p = Pose3d(0.1, 1.2, 2.3, 0.01, 0.1, 1.0)
+        self.assertEqual(str(p), "0.1 1.2 2.3 0.01 0.1 1")
 
     def test_mutable_pose(self):
         pose = Pose3d(0, 1, 2, 0, 0, 0)

